pkgs/containerd/containerd/registry.yaml

```diff
@@ -235,7 +235,7 @@ packages:
         supported_envs:
           - linux
           - windows
-      - version_constraint: semver("<= 1.7.23")
+      - version_constraint: semver("< 2.0.0")
```

Comment:
containerd apparently keeps updating the 1.7.x stream too, so this seems to be in order to cover for that, and the 1.7.23 pkg.yaml entry covers it. https://github.com/containerd/containerd/releases

Reply:
I confirmed v1.7.27 can be installed properly.

```yaml
  - name: containerd/containerd
    version: v1.7.27
```